packages/terra-tabs/src/terra-dev-site/test/tabs/Tabs/TabContentTemplate.scss

```diff
@@ -0,0 +1,6 @@
+:local {
```

Comment:
here also we can move this to single scss file.

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.